playbooks/roles/openconnect/vars/main.yml

```diff
@@ -16,31 +16,42 @@ ocserv_dependencies:
   - libseccomp-dev
   - libwrap0-dev
   - pkg-config
-ocserv_checksum: "sha256:93055ec3b889da680a79c4a51f0efc1b84255e8e2b9c642c5ed1a6f07425b259"
-ocserv_version: "0.11.7"
+
+ocserv_gpg_key_id:            "0x763712747F343FA7"
+ocserv_gpg_key_fingerprint:   "Key fingerprint = 56EE 7FA9 E817 3B19 FE86  268D 7637 1274 7F34 3FA7"
+ocserv_version:               "0.11.8"
 ocserv_compilation_directory: "/usr/local/src/ocserv-{{ ocserv_version }}"
-ocserv_source_filename: "ocserv-{{ ocserv_version }}.tar.xz"
+ocserv_source_filename:       "ocserv-{{ ocserv_version }}.tar.xz"
+ocserv_sig_filename:          "{{ ocserv_source_filename }}.sig"
+ocserv_sig_extension:         "sig"
+mit_gpg_keyserver_address:    "x-hkp://pgp.mit.edu"
```

Comment:
Does this need to be specified or is the key present in [the keyserver we set in the common vars](https://github.com/jlund/streisand/blob/master/playbooks/group_vars/all#L5) ?